backend/app/api.py

```diff
@@ -175,10 +175,30 @@ async def get_movie_from_id(movie_id: int, country_code: str = "DK") -> Movie:
         "&append_to_response=watch/providers,recommendations,credits"
     )
 
+    jw_url = f"http://apis.justwatch.com/content/titles/en_{country_code}/popular"
+
     async with httpx.AsyncClient(http2=True) as client:
         response = await client.get(url)
         movie = response.json()
 
+        jw_res = await client.post(url=jw_url, json={"query": movie.get("title")})
+        jw_providers = jw_res.json()
```

Comment:
It would be great if we did these fetches at the same time - Should be a nice speed-up win